pkg/sql/drop_cascade.go

```diff
@@ -73,9 +73,11 @@ func (d *dropCascadeState) resolveCollectedObjects(
 			tree.ObjectLookupFlags{
 				// Note we set required to be false here in order to not error out
 				// if we don't find the object.
-				CommonLookupFlags: tree.CommonLookupFlags{Required: false},
-				RequireMutable:    true,
-				IncludeOffline:    true,
+				CommonLookupFlags: tree.CommonLookupFlags{
+					Required:       false,
+					IncludeOffline: true,
+					RequireMutable: true,
```

Comment:
super-minor nit: for consistency all other calls to this have require and requiremutable together. I missed this one in my initial diff.

Reply:
Done.

Reply:
Done.